chpl/chpl-api/e2e/script/newman-run.js

```diff
@@ -32,12 +32,37 @@ const apiKeyControllerTests = {
     },
 };
 
-const certifiedProductControllerTests = {
+const certifiedProductControllerDataTests2015listings = {
   ...commonOptions,
   collection: collection_path + '/certified-product-controller.postman_collection.json',
+  folder: 'GET /certified_products- 2015 listings',
+  iterationData: data_path + '/2015chplids-test-data.json',
   reporter: {
     junit: {
-      export: reports_path + '/certified-product-controller-tests.xml',
+      export: reports_path + '/certified-product-controller-data-tests1.xml',
+    },
+  },
+};
+
+const certifiedProductControllerDataTests2014listings = {
```

Comment:
Once above suggested changes (combining 2014 and 2015 listing test and removing separate folders) will be implemented, 'certifiedProductControllerDataTests2014listings ' and 'certifiedProductControllerDataTests2015listings' can be removed from this file

Reply:
This is done